README.md

```diff
@@ -1,181 +1,248 @@
-[npm-badge]: https://img.shields.io/npm/v/oxlint/latest?color=brightgreen
-[npm-url]: https://www.npmjs.com/package/oxlint/v/latest
-[crates-badge]: https://img.shields.io/crates/v/oxc_parser.svg
-[crates-url]: https://crates.io/crates/oxc_parser
-[docs-badge]: https://docs.rs/oxc/badge.svg
-[docs-url]: https://docs.rs/oxc
-[code-size-badge]: https://img.shields.io/github/languages/code-size/web-infra-dev/oxc
-[code-size-url]: https://github.com/web-infra-dev/oxc
-[code-coverage-badge]: https://codecov.io/github/web-infra-dev/oxc/branch/main/graph/badge.svg
-[code-coverage-url]: https://codecov.io/gh/web-infra-dev/oxc
-
-The Oxidation Compiler is creating a suite of high-performance tools for the JavaScript / TypeScript language re-written in Rust:
+## ⚓ Oxc
 
-* [AST](./crates/oxc_ast) - See [docs.rs][docs-ast]
-* [Parser](./crates/oxc_parser) ([acorn][acorn]) - See [docs.rs][docs-parser], [@oxidation-compiler/napi][npm-napi]
-* [Linter](./crates/oxc_linter) ([ESLint][ESLint]) - Prototype - Try it out! `npx oxlint@latest path`
-* Formatter ([prettier][prettier])
-* Transpiler ([babel][babel])
-* [Minifier](./crates/oxc_minifier) ([terser][terser]) - [Prototype](https://github.com/web-infra-dev/oxc/tree/main/crates/oxc_minifier)
-* [Resolver](./crates/oxc_resolver) ([enhanced-resolve](enhanced-resolve))
-* Type Checker - See [ezno][ezno], available via `npx oxidation-compiler@latest check path`
+The Oxidation Compiler is creating a suite of high-performance tools for JavaScript and TypeScript.
 
-[docs-ast]: https://docs.rs/oxc/latest/oxc/ast/index.html
-[docs-parser]: https://docs.rs/oxc/latest/oxc/parser/index.html
-[npm-napi]: https://www.npmjs.com/package/@oxidation-compiler/napi
-[acorn]: https://github.com/acornjs/acorn
-[babel]: https://babel.dev
-[prettier]: https://prettier.io
-[ESLint]: https://eslint.org/
-[prettier]: https://github.com/prettier/prettier
-[ezno]: https://github.com/kaleidawave/ezno
-[terser]: https://terser.org
-[enhanced-resolve]: https://github.com/webpack/enhanced-resolve
+Oxc is a parser, linter, formatter, transpiler, minifier, resolver ... all written in Rust.
 
-## Philosophy
+## 💡 Philosophy
 
-This project follows philosophies from the [Rome](https://rome.tools) and [Ruff](https://beta.ruff.rs) projects.
+This project adheres to philosophies from [Rome][rome] and [Ruff][ruff].
 
 1. JavaScript tooling could be rewritten in a more performant language
 2. An integrated toolchain can tap into efficiencies that are not available to a disparate set of tools
 
-## Milestone
+## ⚡️ Quick Start
 
-This project currently has a fully working parser, a prototype for the linter as well as the minifier.
+The linter is ready to use. It comes with over 60 default rules and no configuration is required.
 
-The current objectives are:
+To start using, install [oxlint][npm-oxlint] or via `npx`:
 
-* Publish the linter as a product
-* Finish the minifier
+```bash
+npx oxlint@latest .
```

Comment:
Feature request: Get rid of the `.` argument :smile: 
(Happy to discuss in a more appropriate place :wink:)

Reply:
This is an interesting approach, with the dot currently gives you a help message, which is a standard CLI user experience?

Reply:
In my (personal) opinion, you should not have to specify which folders the tool should analyze. The tool should either deduce it from files (a `package.json` or a TS config), or it should be configured in the tool's own configuration file. 

I don't think the first option is do-able for JavaScript (maybe with a TS config?), so probably best done in oxlint's configuration file. If users want, they can specify folders to analyze `npx oxlint src/ tests/` but IMO it shouldn't be necessary.

(Btw I'm happy to let you inherit some philosophies from `elm-review` too :stuck_out_tongue:)